src/include/uapi/abi.h

```diff
@@ -55,7 +55,7 @@
 	)
 
 #define SOF_ABI_MAJOR 1
-#define SOF_ABI_MINOR 0
+#define SOF_ABI_MINOR 1
```

Comment:
No need tfo this atm, I'm doing a ABI update so will be bumping major. Lets include this in the ABI update too.

Reply:
withdrawn changes